From owner-linux-pci@atrey.karlin.mff.cuni.cz Tue Oct 10 07:01:53 2006
From: Benjamin Herrenschmidt <benh@kernel.crashing.org>
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: linux-pci@atrey.karlin.mff.cuni.cz, Matthew Wilcox <matthew@wil.cx>
Subject: PCI: Delete unused extern in powermac/pci.c
Date: Tue, 10 Oct 2006 08:01:22 -0600
Message-Id: <1160488883958-git-send-email-matthew@wil.cx>

From: Benjamin Herrenschmidt <benh@kernel.crashing.org>

This file no longer uses pci_cache_line_size, so delete the declaration

Signed-off-by: Matthew Wilcox <matthew@wil.cx>
Acked-by: Jeff Garzik <jeff@garzik.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/powerpc/platforms/powermac/pci.c |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/arch/powerpc/platforms/powermac/pci.c
+++ gregkh-2.6/arch/powerpc/platforms/powermac/pci.c
@@ -48,7 +48,6 @@ static struct pci_controller *u3_ht;
 static int has_second_ohare;
 #endif /* CONFIG_PPC64 */
 
-extern u8 pci_cache_line_size;
 extern int pcibios_assign_bus_offset;
 
 struct device_node *k2_skiplist[2];
